PyRDF/backend/Dist.py

```diff
@@ -646,13 +646,9 @@ def reducer(mergeables_out, mergeables_in):
             self.friend_info = self._get_friend_info(self.tree)
 
         if not self.nentries:
-            # Fall back to local execution
-            # if 'nentries' is '0'
-            msg = ("No entries in the Tree, falling back to local execution!")
-            warnings.warn(msg, UserWarning, stacklevel=2)
-            PyRDF.use("local")
-            from .. import current_backend
-            return current_backend.execute(generator)
+            # Empty trees cannot be processed distributedly
+            raise RuntimeError(
```

Comment:
What does plain `RDataFrame` do when the tree is empty?

Reply:
```
root [0] ROOT::RDataFrame df{"NOMINAL","emptytree.root"};
root [1] auto h = df.Define("x","rdfentry_").Histo1D("x");
root [2] h->Draw()
Info in <TCanvas::MakeDefCanvas>:  created default TCanvas with name c1
Error in <TCanvas::Range>: illegal world coordinates range: x1=0.000000, y1=-0.131250, x2=0.000000, y2=1.181250
Error in <TCanvas::RangeAxis>: illegal axis coordinates range: xmin=0.000000, ymin=0.000000, xmax=0.000000, ymax=1.050000
root [3] auto c = df.Count();
root [4] c.GetValue()
(const unsigned long long) 0
```

Reply:
So it returns an empty histogram and count is 0. What's the price we would need to pay to keep the same behaviour? Construct a plain RDataFrame here and run the graph locally?

Reply:
Yes we'd need to construct a ROOT. RDataFrame . As discussed, let's keep the error for now as this is a safety mechanism and not a feature. Would be easy to change also after the integration in ROOT